### ISSUE TITLE
More translatable messages.

### ISSUE THREAD

**Issue by Acenox** [PeachesMLG/CustomEnchants#7]
Hello,

can you add the translation for "Helmet Enchant" "Tools enchant" etc in the config please.

https://www.noelshack.com/2018-18-7-1525631319-photo-2018-05-06-20-28-14.jpg
https://www.noelshack.com/2018-18-7-1525631321-photo-2018-05-06-20-28-24.jpg
https://www.noelshack.com/2018-18-7-1525631323-photo-2018-05-06-20-28-29.jpg

Best regards,

**Comment by PeachesMLG** [comment]
Yes, Sorry ive been meaning to do this for ages, just never got round to it

**Comment by PeachesMLG** [comment]
please note, i do intend on adding this, I am just fixing all the bugs first

**Comment by Acenox** [comment]
Great ! Thank you :)
